server/boop-secrets.js

```diff
@@ -0,0 +1,44 @@
+const logger = require("./logger");
+
+const yaml = require("js-yaml");
+const fs   = require("fs");
+
+const dotenv = require("dotenv");
+dotenv.config();
+
+const makeYamlSecrets = () => {
+    try {
+        const secrets = yaml.dump(
+            {
+                apiVersion: "v1",
+                kind: "Secret",
+                metadata: {
+                    name: "boop-secrets"
+                },
+                stringData: {
+                    GOOGLE_CLIENT_ID: process.env.GOOGLE_CLIENT_ID,
+                    GOOGLE_CLIENT_SECRET: process.env.GOOGLE_CLIENT_SECRET,
+                    TOKEN_SECRET: process.env.TOKEN_SECRET,
+
+                    DB_NAME: process.env.DB_NAME,
+                    DB_USERNAME: process.env.DB_USERNAME,
+                    DB_PASSWORD: process.env.DB_PASSWORD,
+                    DB_HOST: process.env.DB_HOST,
+                    DB_PORT: Number(process.env.DB_PORT),
+                }
+            },
+            {
+
+            }
+        );
+
+        fs.writeFile("boop-secret.yaml", secrets, function (err) {
```

Comment:
Other components in the project expect the `boop-secret.yaml` file to be in the root of the project e.g. the deploy-script.bash and .gitignore